### PR TITLE
New brain damage string(s)

### PR DESCRIPTION
**Pull request by Ragolution**
@silicons 

## About The Pull Request

Adds sum new Brain Damage string.

## Why It's Good For The Game

It makes me smiel.

## Changelog
:cl:
add: new brain damage trauma string
/:cl:

### CODE REVIEW

**Review by LetterN — CHANGES_REQUESTED** [review]
Spaces on jsons, not tabs.

File: strings/traumas.json [hunk 1]
```diff
@@ -176,13 +178,17 @@
...
+	   "colon marhens",
+	   "haylo",
+        "EEGEE",
         "mr terry",
-        "berry"
+        "berry",
+        "shitterdel",
+        "sitadele",
+        "main",
+        "rp"
```
Comment (LetterN) [inline strings/traumas.json:191]: arrpee?

**Review by ghost — CHANGES_REQUESTED** [review]
M O R E

**Comment by Ragolution** [comment]
> Spaces on jsons, not tabs.

I am 900% sure I did spaces

**Pull request closed without merging**